tests/Test.ADAL.NET.Unit/UnitTests.cs

```diff
@@ -160,6 +160,13 @@ public void AuthenticationParametersTest()
             RunAuthenticationParametersNegative("Bearer authorization_uri=abc=de");
         }
 
+        [TestMethod]
+        [Description("Test for Adal version creation with Regex")]
+        public void AdalVersionRegexTest()
+        {
+            string adalVersion = AdalIdHelper.GetAdalVersion();
+            Assert.AreEqual("0.0.0.0", adalVersion);
```

Comment:
curious what the default version is "0.0.0.0".

Reply:
This caused a problem w/our tests, so I'm just going to assert that it's not null